packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.slow-spec.ts

```diff
@@ -255,6 +255,19 @@ function compareSearchResults(expected: SearchInWorkspaceExpectation[], actual:
 describe('ripgrep-search-in-workspace-server', function (): void {
     this.timeout(10000);
 
+    // Ignoring search with query provided.
+    it('ignore files with ignore query as "potatoes', done => {
+        const pattern = 'sweet potatoes';
+
+        const client = new ResultAccumulator(() => {
+            const expected: SearchInWorkspaceExpectation[] = [];
+            compareSearchResults(expected, client.results);
+            done();
+        });
+        ripgrepServer.setClient(client);
+        ripgrepServer.search(pattern, [rootDirAUri, rootDirBUri], { ignoredFiles: ['potatoes'] });
```

Comment:
We are missing test-cases which I believe should be covered if we want better test coverage of the functionality as to limit potential future bugs, namely:

**Test**: Verifying that multiple ignore globs are supported.

**Test:** Overriding the default `files.exclude` preference value:

Ex:

```json
"files.exclude": {
  "**/.git": false
}
```

**Test:** It is possible that `search.exclude` overrides a default `files.exclude` preference:

Ex:

```json
"search.exclude": {
  "**/.git": false
}
```

**Test:** Combination of preference values (both `true` and `false`):

Ex:

```json
"search.exclude": {
  "**/.md": false,
  "**/.ts": true
}
```

All these tests can be in their own `describe('ignoredFiles', () => {})` section for better readability.